src/components/PageApplicationComments/PageApplicationComments.tsx

```diff
@@ -76,25 +77,33 @@ export const PageApplicationComments = ({
         <h1 className="govuk-heading-l">
           {type === "public" ? "Public Comments" : "Specialist Comments"}
         </h1>
-        <form
-          className="govuk-form"
-          method="get"
-          action={createPathFromParams(params, "comments/search")}
-          aria-label="Search & Sort comments"
-        >
-          <input type="hidden" name="type" value={type} />
-          <input type="hidden" name="council" value={councilSlug} />
-          <input type="hidden" name="reference" value={reference} />
-          <FormCommentsSearch searchParams={searchParams} />
-          <FormCommentsSort searchParams={searchParams} />
-        </form>
+        {pagination?.totalAvailableItems &&
+        pagination?.totalAvailableItems > 0 ? (
+          <form
+            className="govuk-form"
+            method="get"
+            action={createPathFromParams(params, "comments/search")}
+            aria-label="Search & Sort comments"
+          >
+            <input type="hidden" name="type" value={type} />
+            <input type="hidden" name="council" value={councilSlug} />
+            <input type="hidden" name="reference" value={reference} />
+            <FormCommentsSearch searchParams={searchParams} />
+            <FormCommentsSort searchParams={searchParams} />
+          </form>
+        ) : (
+          <></>
+        )}
+
         <hr className="govuk-section-break govuk-section-break--l govuk-section-break--visible"></hr>
         {comments && comments.length > 0 ? (
           <>
             {comments.map((comment) => (
               <CommentCard key={comment.id} comment={comment} />
             ))}
           </>
+        ) : pagination?.totalAvailableItems === 0 ? (
+          <ContentNotPublished type={type} />
```

Comment:
I don't think theres a need for a separate component here? Since it’s a single line with no shared logic or reuse expected, keeping it inline on the page keeps things simpler and easier to follow.